csrankings-l.csv

```diff
@@ -752,6 +753,7 @@ Lu Wang 0004,Texas State University,https://cs.txstate.edu/accounts/profiles/tsu
 Lu Wang 0008,University of Michigan,https://web.eecs.umich.edu/~wangluxy,uczqEdUAAAAJ
 Lu Zhang 0021,University of Arkansas,http://www.csce.uark.edu/~lz006,06ms8fMAAAAJ
 Lu Zhang 0023,Peking University,http://sei.pku.edu.cn/~zhanglu,JUnz2VcAAAAJ
+Luana Ruiz,Johns Hopkins University,https://luanaruiz9.github.io/,SWVeT4AAAAAJ

```

Comment:
<img width="1236" alt="Screen Shot 2024-10-26 at 11 25 22 AM" src="https://github.com/user-attachments/assets/6e568d9d-362b-4550-95fa-ab6a9b5b865e">

On this page: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/